### PR TITLE
Set up automated release workflow

### PR DESCRIPTION
**Pull request by kang-makes**
Setting up automated release workflow.

### CODE REVIEW

File: .github/workflows/automated_release.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  workflow_dispatch:
+  schedule:
+    - cron: "0 10 * * 4"
+
+jobs:
+  release_management:
+    uses: newrelic/coreint-automation/.github/workflows/reusable_release_automation.yaml@v3
+    secrets: inherit
+    with:
+      rt-included-files: bundle.yml
```
Comment (paologallinaharbur) [inline .github/workflows/automated_release.yaml:13]: Dockerfile?

**Review by paologallinaharbur — APPROVED** [review]

File: .github/workflows/automated_release.yaml (hunk 1 above)
Comment (kang-makes) [reply .github/workflows/automated_release.yaml:13]: That is for development purposes.

`agent_version` on the `Dockerfile` is set to latest but it is a build-arg.

The image is build here: https://github.com/newrelic/infrastructure-bundle/blob/d8d2091729d589bde7820347d33319e15eb7a947/.github/workflows/release.yml#L43-L47

The agent version is got from the downloader: https://github.com/newrelic/infrastructure-bundle/blob/d8d2091729d589bde7820347d33319e15eb7a947/docker-build.sh#L10-L17

The downloader finally get it from the build.yml: https://github.com/newrelic/infrastructure-bundle/blob/d8d2091729d589bde7820347d33319e15eb7a947/downloader.go#L72-L82

**Review by marcsanmi — APPROVED** [review]

File: .github/workflows/automated_release.yaml (hunk 1 above)
Comment (kang-makes) [reply .github/workflows/automated_release.yaml:13]: Renovate is already doing the magic: https://github.com/newrelic/infrastructure-bundle/blob/d8d2091729d589bde7820347d33319e15eb7a947/.github/renovate.json5#L12-L22

Comment (paologallinaharbur) [reply .github/workflows/automated_release.yaml:13]: awesome! thanks!

**Pull request merged**